recipes/proovframe/meta.yaml

```diff
@@ -12,8 +12,10 @@ source:
   sha256: '{{ sha256 }}'
 
 build:
-  number: 0
+  number: 1
   noarch: generic
+  run_exports:
+    - {{ pin_subpackage('proovframe', max_pin='x.x.x') }}
```

Comment:
Can you check https://bioconda.github.io/contributor/linting.html?highlight=max_pin#missing-run-exports and confirm you wanted to set max_pin with `x.x.x`?

Reply:
Yes I think it is fine, from what I see you follow the standard semantic versioning